test/unit/chainlinkOperator/ChainlinkOperator.behavior.ts

```diff
@@ -1,13 +1,13 @@
 import shouldBehaveLikeDeleteFeed from "./effects/deleteFeed";
 import shouldBehaveLikeSetFeed from "./effects/setFeed";
-import shouldBehaveLikeGetAdjustedPrice from "./view/getAdjustedPrice";
+import shouldBehaveLikegetNormalizedPrice from "./view/getNormalizedPrice";
 import shouldBehaveLikeGetFeed from "./view/getFeed";
 import shouldBehaveLikeGetPrice from "./view/getPrice";
 
 export function shouldBehaveLikeChainlinkOperator(): void {
   describe("View Functions", function () {
-    describe("getAdjustedPrice", function () {
-      shouldBehaveLikeGetAdjustedPrice();
+    describe("getNormalizedPrice", function () {
```

Comment:
Can we arrange these describe blocks alphabetically?

Reply:
done